x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -181,10 +181,12 @@ qualifiedNamePatterns
 identifier
     : UNQUOTED_IDENTIFIER
     | QUOTED_IDENTIFIER
+    | params
```

Comment:
params implies plural, aka a list. Yet here it's just one - so should be param. If it clashes with PARAM in the lexer, rename it to something else such as parameter.